hugging_face/hugging_face_pipeline_for_monai.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>Missing remove tempdir in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>